privacy-on-beam/pbeam/sum_test.go

```diff
@@ -305,6 +408,38 @@ func TestSumPerKeyCrossPartitionContributionBoundingInt(t *testing.T) {
 	}
 }
 
+// // Checks that SumPerKey with partitions bounds cross-partition contributions correctly with int values.
+func TestSumPerKeyWithPartitionsCrossPartitionContributionBoundingInt(t *testing.T) {
+	// triples contains {1,0,1}, {2,0,1}, …, {50,0,1}, {1,1,1}, …, {50,1,1}, {1,2,1}, …, {50,9,1}.
+	var triples []tripleWithIntValue
+	for i := 0; i < 10; i++ {
+		triples = append(triples, makeDummyTripleWithIntValue(50, i)...)
+	}
+	result := []testInt64Metric{
+		{0, 150},
+	}
+	p, s, col, want := ptest.CreateList2(triples, result)
+	col = beam.ParDo(s, extractIDFromTripleWithIntValue, col)
+	partitionsCol := CreateList(s, []int{0, 1, 2})
```

Comment:
We are not actually testing cross-partition contribution bounding if we have a total of 3 kept partitions. We should keep more partitions and let Privacy on Beam drop the extra ones. So, let's say you keep `{0, 1, 2, 3, 4}`.

Reply:
Same as previous comments. Testing that bound is not wasted on unspecified partitions.